webpack.dev.js

```diff
@@ -1,20 +1,27 @@
 const DuplicatePackageCheckerPlugin = require('duplicate-package-checker-webpack-plugin');
 const FriendlyErrorsWebpackPlugin = require('friendly-errors-webpack-plugin');
-const sso = require('./server/sso');
+const Sso = require('./server/sso');
 const serverProxy = require('./server/proxy');
 
-const proxyOptions = {
-  target: 'https://www.ovh.com',
-  context: ['/engine', '/auth'],
-  changeOrigin: true,
-  logLevel: 'silent',
+const TARGET = {
+  eu: 'https://www.ovh.com',
+  ca: 'https://ca.ovh.com',
+  us: 'https://us.ovhcloud.com',
 };
 
 module.exports = (env) => {
+  const region = (env.region || 'eu').toLowerCase();
+  const proxyOptions = {
```

Comment:
We have created `/server/proxy/aapi` to isolate config, maybe we should move this part in `/server/proxy/***` .

Reply:
yep i'm going to move it